### PR TITLE
Bugfix/category mapping

### PR DESCRIPTION
**Pull request by antazoey**

### CODE REVIEW

File: Packs/Code42/Integrations/Code42/Code42.py [hunk 1]
```diff
@@ -641,7 +645,7 @@ def _create_file_category_filters(self):
         observed_file_categories = self._observation_data.get("fileCategories")
         if observed_file_categories:
             categories = [
-                c.get("category").upper()
+                _get_file_category_value(c.get("category"))
```
Comment (antazoey) [inline Packs/Code42/Integrations/Code42/Code42.py:648]: bug was here

File: Packs/Code42/Integrations/Code42/Code42_test.py [hunk 2]
```diff
@@ -724,7 +724,7 @@
             },
             {
                 "filterClause": "AND",
-                "filters": [{"operator": "IS", "term": "fileCategory", "value": "IMAGE"}],
+                "filters": [{"operator": "IS", "term": "fileCategory", "value": "SOURCE_CODE"}],
```
Comment (antazoey) [inline Packs/Code42/Integrations/Code42/Code42_test.py:727]: This is the only way I have verified that this fixes the bug

**Comment by antazoey** [comment]
Once py42 releases, we can just use the constants there. This will have to be the fix for now.

And I do have a story in Jira for using py42 constants

File: Packs/Code42/Integrations/Code42/Code42_test.py [hunk 3]
```diff
@@ -802,8 +802,8 @@
                 "filterClause": "AND",
                 "filters": [
                     {"operator": "IS_NOT", "term": "exposure", "value": "IsPublic"},
-                    {"operator": "IS_NOT", "term": "exposure", "value": "OutsideTrustedDomains"},
                     {"operator": "IS_NOT", "term": "exposure", "value": "SharedViaLink"},
+                    {"operator": "IS_NOT", "term": "exposure", "value": "OutsideTrustedDomains"},
```
Comment (alanag13) [inline Packs/Code42/Integrations/Code42/Code42_test.py]: Why was this moved? I think these need to be in alphabetical order due to changes that were made that make the filters get sorted when are is called on them.

Comment (antazoey) [reply Packs/Code42/Integrations/Code42/Code42_test.py]: I am not sure but why, but the tests expect a different order

Comment (antazoey) [reply Packs/Code42/Integrations/Code42/Code42_test.py]: Ah, my bad, my py42 was outdated in that environment

**Review by alanag13 — APPROVED** [review]
